local-cli/util/__mocks__/fs.js

```diff
@@ -0,0 +1,368 @@
+/**
```

Comment:
This is a copy of the fs mock from metro-bundler. Adds `exists`/`existsSync` + `readlink`/`readlinkSync` support.